vlib/v/checker/tests/modify_const_with_ref.out

```diff
@@ -5,10 +5,3 @@ vlib/v/checker/tests/modify_const_with_ref.vv:11:11: error: `constant` is immuta
       |              ^
    12 |     c.value = 200
    13 | }
-vlib/v/checker/tests/modify_const_with_ref.vv:9:6: error: unused variable: `unused_var`
```

Comment:
why remove this test? it's correct.

Reply:
`unused_var = Foo{}`, so `unused_var` is used.

Reply:
no, it's not

it's only used only if the actual var is used, like in Go

this is a recent change I did

Reply:
Ok, I see.